homeassistant/components/usb/flow.py

```diff
@@ -29,14 +29,16 @@ def __init__(self, hass: HomeAssistant) -> None:
     def async_start(self, *_: Any) -> None:
         """Start processing pending flows."""
         self.started = True
-        self.hass.loop.call_soon(self._async_process_pending_flows)
+        self.hass.async_add_job(self._async_process_pending_flows)
```

Comment:
Maybe we don't need to add a job for this at all? Just call `_async_process_pending_flows` since it just creates tasks.

Reply:
Agreed. I don't think it even needs to be its own function.